storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/DefaultResourceAwareStrategy.java

```diff
@@ -39,6 +39,9 @@
 
     @Override
     public SchedulingResult schedule(Cluster cluster, TopologyDetails td) {
+        boolean oneWorkerPerExecutor = (Boolean) td.getConf().getOrDefault(Config.TOPOLOGY_RAS_ONE_WORKER_PER_EXECUTOR, false);
```

Comment:
I don't think you need to set a default here. defaults.yaml should always be loaded, so there's no risk of the property being null.